internal/acceptance/sql_table_test.go

```diff
@@ -264,3 +266,81 @@ func TestUcAccResourceSqlTable_Liquid(t *testing.T) {
 		}`,
 	})
 }
+
+func constructManagedSqlTableTemplate(columnName string, typeName string, nullable bool, comment string) string {
```

Comment:
Suggestion: this will be much more flexible if we define it as
```go
func constructManagedSqlTableTemplate(columns []catalog.SqlColumnInfo) string {
  ...
}
```
It will extend to the next set of tests that cover changing the number of columns.